docs/assets/js/swagger.yml

```diff
@@ -1777,6 +1775,30 @@ paths:
         required: true
         schema:
           type: string
+    get:
```

Comment:
This seems to be off by one, it incorporates a _previous_ change into this swaggacopy but not _this_ change.  Can you regenerate it before pulling?

Reply:
sure - I was sure the makefile validates they are the same.

Reply:
```shell
% sha256sum api/swagger.yml docs/assets/js/swagger.yml
9674aad548414c911e7c7d21f22bae22632e9c6e9f7784875549eb74917c34f4  api/swagger.yml
9674aad548414c911e7c7d21f22bae22632e9c6e9f7784875549eb74917c34f4  docs/assets/js/swagger.yml
```